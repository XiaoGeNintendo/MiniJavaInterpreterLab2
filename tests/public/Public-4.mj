int main() {
    char x = 'G';
    var res = foo(x);
    println(res);

    return 0;
}

int foo(int x) {
    return x + 10;
}

char foo(char x) { // matched
    return (char) (x + 1);
}