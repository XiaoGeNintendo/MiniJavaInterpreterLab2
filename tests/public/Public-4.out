H
Process exits with the code 0.
